charts/qiskit-serverless/charts/gateway/templates/deployment.yaml

```diff
@@ -109,6 +109,10 @@ spec:
               value: {{ .Values.application.auth.token.verificationUrl | quote }}
             - name: SETTINGS_TOKEN_AUTH_VERIFICATION_FIELD
               value: {{ .Values.application.auth.token.verificationField | quote }}
+{{- if .Values.application.authMockproviderRegistry }}
+            - name: SETTINGS_AUTH_MOCKPROVIDER_REGISTRY
+              value: {{ .Values.application.authMockproviderRegistry }}
```

Comment:
I would add a default value in the `values.yaml` like `icr.io` or `test` as you were using, wdyt?